.github/workflows/test.yml

```
@@ -109,3 +114,27 @@ jobs:
           fi
           qmake
         shell: bash
+
+      - name: Install tools with options
+        if: ${{ !matrix.qt.version }}
+        uses: ./
+        with:
+          tools-only: true
+          tools: tools_ifw tools_qtcreator,qt.tools.qtcreator
+          cache: ${{ matrix.cache == 'cached' }}
+
+      - name: Test installed tools
+        if: ${{ !matrix.qt.version }}
+        env:
+          # Conditionally set qtcreator path based on os:
+          QTCREATOR_BIN_PATH: ${{ startsWith(matrix.os, 'macos') && '../Qt/Qt Creator.app/Contents/MacOS/' || '../Qt/Tools/QtCreator/bin/' }}
+        shell: bash
+        run: |
+          # Check if QtIFW is installed
+          ls ../Qt/Tools/QtInstallerFramework/*/bin/
+          ../Qt/Tools/QtInstallerFramework/*/bin/archivegen --version
+  
+          # Check if QtCreator is installed: QtCreator includes the CLI program 'qbs' on all 3 platforms
+          ls "${QTCREATOR_BIN_PATH}"
+          [[ -f "${QTCREATOR_BIN_PATH}qbs" ]] && echo "qbs is installed"
+          "${QTCREATOR_BIN_PATH}qbs" --version || echo "qbs installed but fails to run"
```

Please note: On Ubuntu 18.04, the current version of the `qbs` tool cannot run without a newer version of GLIBC. This script will cause the error message to print in the logs, but it swallows the error to prevent build failure.

I can see four ways to handle this:
1. Leave it as is, and expect the user to add dependencies like newer GLIBC on their own.
2. Just remove the Ubuntu 18.04 builds, and make the action fail if `qbs --version` cannot run (my preference).
3. Modify the workflow to install such dependencies (this will require maintenance every time a new version of QtCreator stops supporting an old Ubuntu).
4. Modify the action to install such dependencies, so a user can't ever get it wrong (high maintenance, will require testing every tool available).

---

I just saw this: https://github.blog/changelog/2022-08-09-github-actions-the-ubuntu-18-04-actions-runner-image-is-being-deprecated-and-will-be-removed-by-12-1-22/

If Ubuntu 18.04 images are deprecated anyway, I think it's probably appropriate to go with option 2.

---

Ubuntu 18 builds removed.